sftp/sftp.go

```
@@ -220,25 +220,31 @@ func (f *Fs) List(out fs.ListOpts, dir string) {
 // Put data from <in> into a new remote sftp file object described by <src.Remote()> and <src.ModTime()>
 func (f *Fs) Put(in io.Reader, src fs.ObjectInfo) (fs.Object, error) {
 	debug( f, "Put '" + src.Remote() + "'")
-	f.mkdir( f.sftpClient.Join( f.root, filepath.Dir( src.Remote() ) ) )
+	_ = f.mkdir( f.sftpClient.Join( f.root, filepath.Dir( src.Remote() ) ) )
```

Don't ignore the error please!